src/Consolonia.Core/Drawing/PixelBufferImplementation/BoxPattern.cs

```diff
@@ -86,8 +52,8 @@ private static char GetBoxSymbol(byte upRightDownLeft)
                 {
                     return upRightDownLeft switch
                     {
-                        EmptySymbol => char.MinValue,
-                        BoldSymbol => '█',
+                        EmptyMask => char.MinValue,
```

Comment:
Similar about this. How it works it does not look like a mask, it looks like a value.